### DISCUSSION TITLE
Hilo de fechas de presentación

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-IdSw1#130] [Announcements]
Por favor, indicar como respuesta a este hilo las fechas que desean para la revisión para de sus trabajos. Recordar, como vimos en clase, que podíamos hacer la revisión antes del 20 de diciembre o ya luego, en enero.

**Comment by TheMoys** [comment]
A mí grupo le gustaría presentar el día 20 de diciembre.

**Comment by natiicruuz** [comment]
A mi grupo en enero, quedamos por definir la fecha exacta.

**Reply by mmasias** [reply]
De acuerdo. Si es enero, muy probablemente sea el día del examen final.

**Reply by natiicruuz** [reply]
@mmasias 
Nosotros presentamos el jueves 16
Saludos, feliz año

**Reply by mmasias** [reply]
> @mmasias Nosotros presentamos el jueves 16 Saludos, feliz año

Por favor, crear una entrada en Google Calendar en el que estén todos los integrantes del grupo y yo.

**Reply by mmasias** [reply]
He ajustado la hora, porque antes había solicitado otro grupo y la duración prevista de las exposiciones es de 2 horas.

**Comment by 0xJVR** [comment]
Grupo 5, Enero.

**Comment by miguelgomez75** [comment]
A nuestro grupo (el 6) en enero

**Reply by miguelgomez75** [reply]
@mmasias Sería posible presentar el día 17 de enero a las 10:30? ya he creado el evento en calendar

**Reply by mmasias** [reply]
Ok al día, pero empezamos a las 9:30h.

**Comment by mmasias** [comment]
Quedan actualizados en el [README.md](https://github.com/mmasias/24-25-IdSw1-SDR/blob/main/README.md) del repo. Los grupos restantes, notificar por esta vía.

**Comment by celiabecerril** [comment]
El grupo 1 lo presentaremos en enero.

**Comment by DavidGarciaCosta** [comment]
Grupo de Susana en Enero

**Comment by aadrigaar** [comment]
El grupo de David García en enero

**Comment by TheMoys** [comment]
Buen día @mmasias,

Como bien te comentamos hoy por la mañana, estaríamos entregandote el proyecto el viernes 17 de enero.
